libs/skills/catalog/create-tool/references/file-layout.md

```
@@ -0,0 +1,96 @@
+---
+name: file-layout
+description: Flat sibling vs folder-per-tool layouts. <name>.schema.ts / <name>.tool.ts / <name>.tool.spec.ts.
+---
+
+# File layout for tools
+
+Two endorsed layouts. Pick based on tool count per app and whether each tool has local helpers / fixtures / error types.
+
+## Flat siblings (≤3 tools per app, or each tool fits in one screen)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)